tests/accessibility_tests/common_steps/accessibility.py

```diff
@@ -0,0 +1,31 @@
+
+import re
+from playwright.sync_api import Page
+from axe_core_python.sync_playwright import Axe
+import json
+import pytest
+import os
+
+@pytest.fixture()
+def test_accessibility(page: Page):
+    # Ensures run after test
+    yield
```

Comment:
We are yielding here to ensure the first call does nothing, however, don't we want to initialise Axe before the tests then output the results at the end?

Take a look at Mozilla's implementation here: [https://github.com/mozilla-services/pytest-axe/blob/master/pytest_axe/tests/conftest.py](https://github.com/mozilla-services/pytest-axe/blob/master/pytest_axe/tests/conftest.py)

This may be fine, however, I am unsure what would happen if your test span multiple pages. I think it may just check the last one.